src/frontend/models/Article.ts

```diff
@@ -0,0 +1,7 @@
+export class Article {
+    constructor(public id: number, public name: string, public group: number) {
```

Comment:
Folgendes is redundant, wenn `public`/`private `im constructor angegeben wird:
```
this.id = id;
this.name = name;
this.group = group;
```
Hier ein Beispiel: http://tinyurl.com/z26qhrt